### DISCUSSION TITLE
Error when installing

### DISCUSSION THREAD

**Discussion by WikWikLMU** [AcademySoftwareFoundation/OpenTimelineIO#1372] [Q&A]
"CMake Error: CMAKE_C_COMPILER not set, after EnableLanguage
      CMake Error: CMAKE_CXX_COMPILER not set, after EnableLanguage
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for opentimelineio
Failed to build opentimelineio
ERROR: Could not build wheels for opentimelineio, which is required to install pyproject.toml-based projects"

How do I fix this??

**Comment by JeanChristopheMorinPerso** [comment]
Hi, can you tell us more about the context please? What OS and version (and architecture) do you use? Which python version? What command did you use?

**Comment by jminor** [comment]
@WikWikLMU do you still need help installing OpenTimelineIO?
